velox/type/parser/TypeParser.yy

```diff
@@ -31,7 +31,7 @@
     #include <velox/type/parser/Scanner.h>
     #define yylex(x) scanner->lex(x)
     using namespace facebook::velox;
-    TypePtr typeFromString(const std::string& type) {
+    TypePtr typeFromString(const std::string& type, bool failIfNotRegistered = true) {
```

Comment:
Can we move this code into an .h/.cpp file and call it from here. It is hard to read the code as part of .yy file.

Reply:
done